theme/models.py

```diff
@@ -266,13 +268,15 @@ class UserProfile(models.Model):
         blank=True,
         help_text="The name of the organization you work for."
     )
-    phone_1 = models.CharField(max_length=1024, null=True, blank=True)
+    phone_regex = RegexValidator(regex=r'^\d{8,15}$',
+                                 message="Phone number must be entered in the format: '999999999'. Up to 16 digits allowed.")
+    phone_1 = models.CharField(validators=[phone_regex], max_length=16, null=True, blank=True)
```

Comment:
I believe this requires a model migration. You can run ```docker exec -ti hydroshare python manage.py makemigrations``` if you have not already which should create a new migration file for this change. Then you can check in the migration file.

Reply:
Yes, it requires migration. Thanks for the command! I just checked in my file.